packages/extension-vscode/package.json

```diff
@@ -0,0 +1,73 @@
+{
+    "activationEvents": [
+        "onLanguage:css",
+        "onLanguage:html",
+        "onLanguage:javascript",
+        "onLanguage:json",
+        "onLanguage:jsonc"
+    ],
+    "ava": {
+        "failFast": false,
+        "files": [
+            "dist/tests/**/*.js"
+        ],
+        "timeout": "1m"
+    },
+    "dependencies": {
+        "vscode": "^1.1.21",
+        "vscode-languageclient": "^5.0.1",
+        "vscode-languageserver": "^5.0.3"
+    },
+    "description": "Run webhint in Visual Studio Code.",
+    "devDependencies": {
+        "ava": "^0.25.0",
+        "cpx": "^1.5.0",
+        "eslint": "^5.6.0",
+        "eslint-plugin-import": "^2.14.0",
+        "eslint-plugin-markdown": "^1.0.0-beta.7",
+        "eslint-plugin-typescript": "^0.12.0",
+        "hint": "^3.4.0",
+        "markdownlint-cli": "^0.13.0",
+        "npm-link-check": "^2.0.0",
+        "npm-run-all": "^4.1.2",
+        "nyc": "^13.0.1",
+        "rimraf": "^2.6.2",
+        "typescript": "^3.0.3",
```

Comment:
You are using `proxyquire` later on but I don't see it here. Also we have a pinned version everywhere for `2.0.0` and use the `greenkeeper` property in the `package.json` because otherwise tests don't pass everywhere and haven't had the time to investigate.

Reply:
I don't see `sinon` neither.

Reply:
Good catch! I think I stumbled into these working because of yarn workspaces. I'll get the explicit dependencies added.